core/lib/components/balena/sdk.js

```diff
@@ -20,15 +20,18 @@ const map = require('lodash/map');
 const pick = require('lodash/pick');
 const find = require('lodash/find');
 const flatMapDeep = require('lodash/flatMapDeep');
-
+const fs = require('fs')
+const { join } = require("path");
 const Bluebird = require('bluebird');
 const retry = require('bluebird-retry');
-
 const utils = require('../../common/utils');
 const exec = Bluebird.promisify(require('child_process').exec);
+// const fse = require('fs-extra')
```

Comment:
Are these supposed to be here still?

Reply:
These imports are for the caching code that is present, if we want to not merge the caching code as comments then I can remove the imports as well. I am just making sure we don't forget caching but leave the imports in there for no good reason.